testcases/OpTestEEH.py

```diff
@@ -207,12 +241,26 @@ def check_eeh_slot_resets(self):
     # @returns True/False @type boolean
     #
     def check_eeh_pe_recovery(self, pe):
-        list = self.get_list_of_pci_devices()
-        for device in list:
-            if pe in device:
-                return True
-        return False
-
+     #   cmd = "dmesg  | grep -i 'iommu: Removing device'; echo $?"
+     #   tries = 30
+     #   for i in range(1, tries+1):
+     #       res = self.cv_SYSTEM.sys_get_ipmi_console().run_command(cmd)
+     #       if int(res[-1]):
+     #           print "Waiting for PE %s removal: (%d/%d)" % (pe, i, tries)
+     #           time.sleep(1)
+     #       else:
+	 #		    break
+     #   else:
+     #       raise EEHRemoveFailed("PE", pe)
+        tries = 60
+        for j in range(1, tries+1):
+            list = self.get_list_of_pci_devices()
+            for device in list:
+                if pe in device:
+                    return True
+            time.sleep(1)
+        else:
+            return False
```

Comment:
why switch from exceptions to returning true/false?

Exceptions let us attach detail as to what happened, true/false does not.